### ISSUE TITLE
openbsd64 box breaks with rsync not found

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#1695]
```
$ vagrant up openbsd64
...
    openbsd64: Inserting generated public key within guest...
    openbsd64: Removing insecure key from the guest if it's present...
    openbsd64: Key inserted! Disconnecting and reconnecting using new SSH key...
==> openbsd64: Machine booted and ready!
==> openbsd64: Installing rsync to the VM...
==> openbsd64: Rsyncing folder: /home/tw/w/borg/ => /vagrant/borg/borg
There was an error when attempting to rsync a synced folder.
Please inspect the error message below for more info.

Host path: /home/tw/w/borg/
Guest path: /vagrant/borg/borg
Command: "rsync" "--verbose" "--archive" "--delete" "-z" "--no-owner" "--no-group" "--rsync-path" "sudo rsync" "-e" "ssh -p 2222 -o LogLevel=FATAL  -o ControlMaster=auto -o ControlPath=/tmp/ssh.252 -o ControlPersist=10m  -o IdentitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i '/home/tw/w/borg/.vagrant/machines/openbsd64/virtualbox/private_key'" "--exclude" ".vagrant/" "/home/tw/w/borg/" "vagrant@127.0.0.1:/vagrant/borg/borg"
Error: sudo: rsync: command not found
rsync: connection unexpectedly closed (0 bytes received so far) [sender]
rsync error: error in rsync protocol data stream (code 12) at io.c(226) [sender=3.1.1]
```

**Comment by ThomasWaldmann** [comment]
vagrant version: 1.8.4 and 1.8.6 (debian package) on ubuntu linux 64bit.

have seen similar effect before: #716

**Comment by ThomasWaldmann** [comment]
Fixed by #1724.
